spec/services/course_exports/prepare_students_export_service_spec.rb

```diff
@@ -97,7 +97,11 @@
   let!(:school_admin) { create :school_admin, school: school }
 
   let(:course_export) do
-    create :course_export, :students, course: course, user: school_admin.user
+    create :course_export,
+           :students,
+           course: course,
+           user: school_admin.user,
+           cohorts: [cohort_live]
```

Comment:
The spec doesn't check for cohort scoping; Have more than one cohort on the list to ensure that the export works as expected. 

Add students and submissions for the additional cohorts.